watch-w-docker.sh

```diff
@@ -10,11 +10,12 @@ cd $(dirname $0)
 export MSYS_NO_PATHCONV=1
 export MSYS2_ARG_CONV_EXCL="*"
 
+# -e CHOKIDAR_USEPOLLING=true \ --> hot reload, beware of CPU usage
 docker run --rm \
     -p 3000:3000 \
     -v `pwd`:/work \
     -w /work \
-    -e CHOKIDAR_USEPOLLING=true \ #hot reload, beware of CPU usage
+    -e CHOKIDAR_USEPOLLING=true \

```

Comment:
Ohh, I did not see it

Reply:
It was my mistake for not running it first :/ Now I've tested and it works.